lib/response-builder.ts

```diff
@@ -83,6 +83,7 @@ function pluginMetadataRes(
     packageManager: depsAnalysis.packageManager,
     dockerImageId: depsAnalysis.imageId,
     imageLayers: depsAnalysis.imageLayers,
+    rootFs: depsAnalysis.rootLayers,
```

Comment:
why do we sometimes call it rootFs and sometimes rootLayers, while the "formal" name is rootFsLayers? 😆

Reply:
yeah, I was wondering if that wasn't confusing! I wanted to give it a more user friendly name passing it around, rather than "rootFs", so I named it "rootLayers" everywhere and the we just have "rootFs" object on the plugin response, I can rename, don't mind too much :-)

Reply:
I've renamed it to "rootFsLayers"